metadata_utilities.py

```diff
@@ -73,7 +73,7 @@ def _serialize_xml(write, elem, encoding, qnames, namespaces):
 # END MONKEYPATCH CDATA
 
 
-def generate_iso_metadata(project_definition=None):
+def generate_iso_metadata(supplemental_information=None):
```

Comment:
Just wondering about the terminology. If I understand this correctly, you are now calling a JSON object that contains N number of project definitions: 'supplemental information'? This name comes from the GeoNode layer field that is used to house the information, but perhaps we can consider other names that describe the JSON a bit better, like 'project definition container'? Or maybe the JSON is called a 'project definition' and then the blocks (that we are now calling  project definitions) are called something else, like 'project scheme'?